code/modules/mob/living/carbon/xenomorph/castes/queen/abilities_queen.dm

```diff
@@ -226,8 +226,6 @@
 /datum/action/xeno_action/watch_xeno/proc/on_ctrl_middle_click(datum/source, atom/A)
 	SIGNAL_HANDLER
 	var/mob/living/carbon/xenomorph/queen/watcher = owner
-	if(!watcher.check_state())
```

Comment:
Means you can use it while crit unless i am mistaken

Reply:
Addressed in #10282 by placing the `can_use_action()` check inside the start overwatch proc.

The action bar button click method bas been completely removed with that PR.

Reply:
Not sure what the issue is anyway other than the crit xeno not being able to cancel watching if they had accidentally clicked an `(F)` in chat. Gives them no advantage other than more sight maybe? The "crit blindness" might still even apply idk. For sure they would still not be able to communicate.